docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Added strawman proposal to #1695.

BTW, if it makes any of this simpler (and I think it does), we should consider just targeting v1beta3+.